go.mod

```diff
@@ -7,12 +7,12 @@ require (
 	github.com/gizak/termui/v3 v3.1.0
 	github.com/go-ole/go-ole v1.2.4 // indirect
 	github.com/mattn/go-runewidth v0.0.4
-	github.com/nsf/termbox-go v0.0.0-20200418040025-38ba6e5628f1
+	github.com/nsf/termbox-go v0.0.0-20200418040025-38ba6e5628f1 // indirect
```

Comment:
This was manually pinned because newer versions of termbox-go break FreeBSD; see a1bd0cc9eeec86dac382c4b027fc738b96bfa2fe